apps/rigidCubes/Main.cpp

```diff
@@ -54,6 +54,15 @@ int main(int argc, char* argv[]) {
   MyWindow window;
   window.setWorld(myWorld);
 
+  for(size_t i=0; i<myWorld->getNumSkeletons(); ++i)
+  {
+    dart::dynamics::SkeletonPtr skel = myWorld->getSkeleton(i);
+    for(size_t j=0; j<skel->getNumBodyNodes(); ++j)
+    {
+      std::cout << skel->getBodyNode(j)->getSpatialInertia() << "\n" << std::endl;
+    }
+  }
+
```

Comment:
Hahah, I would guess so. Probably from when I created the Spatial Inertia class. I'll remove it.